### PR TITLE
Spec file and updated building documentation

### PR DESCRIPTION
**Pull request by m-blaha**
The spec is synchronized with downstreams, fixed building for Fedora rawhide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,35 +32,30 @@ for documentation build:
 
 *   doxygen http://www.stack.nl/~dimitri/doxygen/
 
+for rpm building:
+
+*   tito https://github.com/dgoodwin/tito
```
Comment (j-mracek) [inline README.md]: Please add build command example

Comment (j-mracek) [reply README.md]: Sorry , please rephrase that. From first look I would understand that as a command to build. Build example command is bellow and it is fine.

**Comment by j-mracek** [comment]
It looks good, only minor change requested.

**Comment by m-blaha** [comment]
@j-mracek sorry, but i can't find where this 'minor change' is specified

File: README.md (hunk 1 above)
Comment (j-mracek) [inline README.md]: This line look like a command and can be confuzing

**Comment by j-mracek** [comment]
Here it is. Not know why it disappeared.

**Comment by j-mracek** [comment]
But as I see it follows style of the rest. ok, no complains at all. Merging

**Pull request merged**